desktop/release-notes.md

```diff
@@ -24,6 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- [mac] Quit Docker Desktop completely from the Dock menu (https://github.com/docker/for-mac/issues/6440)
+- Enable ComposeV2 after factory reset
+- Copied terminal contents do not contain non-breaking spaces anymore
+- Upgrade kernel to 5.10.124
+- Enables ComposeV2 by default on new installations/if there is no previous configuration.
+- Prevents users on ARM macs without Rosetta installed from switching back to ComposeV1 (only has intel binaries).
+- Added View menu with Zoom In, Zoom Out, and Actual Size options to control the size of the Docker Desktop UI.
+- Descending default sort order for volumes size and created column, containers started column
+- Defined sort order for container/compose state column running > some running > paused > some paused > exited > some exited > created
+- Re-organized container row actions
+- Added compose stop button if any related container is stoppable.
+- Individual compose containers are now deletable from overview
+- Runs every command on the Quickstart guide immediately.
+- Remove workaround for virtiofsd <-> qemu protocol mismatch on Fedora 35, as it is no longer needed. Fedora 35 users should upgrade the qemu package to the most recent version (qemu-6.1.0-15.fc35 as of the time of writing).
+- Implemented an integrated terminal for containers.
+- Updated the tooltip design.
+- Show address for external links.
+
+### Upgrades
+
+- [Docker Compose v2.10.2](https://github.com/docker/compose/releases/tag/v2.10.2)
+- [Docker Compose "Cloud Integrations" v1.0.29](https://github.com/docker/compose-cli/releases/tag/v1.0.29)
+- [Docker Scan v0.19.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.19.0)
+- [Kubernetes v1.25.0](https://github.com/kubernetes/kubernetes/releases/tag/v1.25.0)
+- [Go 1.19](https://github.com/golang/go/releases/tag/go1.19)
+- [cri-dockerd v0.2.5](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.5)
+- [Buildx v0.9.1](https://github.com/docker/buildx/releases/tag/v0.9.1)
+- [containerd v1.6.8](https://github.com/containerd/containerd/releases/tag/v1.6.8)
+- [containerd v1.6.7](https://github.com/containerd/containerd/releases/tag/v1.6.7)
+- [runc v1.1.4](https://github.com/opencontainers/runc/releases/tag/v1.1.4)
+- [runc v1.1.3](https://github.com/opencontainers/runc/releases/tag/v1.1.3)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed issues with the image list appearing empty in Docker Desktop even though there are images. Related to [docker/for-win#12693](https://github.com/docker/for-win/issues/12693) and [docker/for-mac#6347](https://github.com/docker/for-mac/issues/6347).
+- Fixed a bug that made Docker clients in some languages hang on `docker exec`. Fixes [https://github.com/apocas/dockerode/issues/534](https://github.com/apocas/dockerode/issues/534).
+- Extensions SDK: prevent failed spawned command to make Docker Desktop unexpectedly quit.
+- Fixes container terminal resizing issues.
```

Comment:
That’s odd because further up we have an entry saying we have just added that as a new feature.
We don’t need to document bug fixes for not yet released features. 😄

Reply:
Oh, right, that is odd to leave in there. Will take it out :D